website/assets/js/blockRedactor.js

```diff
@@ -296,52 +374,71 @@ export class BlockRedactor {
             document.getElementById('wrongbx').hidden = !document.getElementById('wrongbx').hidden;
         };
 
-        const specialState = BlockRedactor.loadAnswers(question, instance);
+        BlockRedactor.loadAnswers(question, instance);
```

Comment:
There are you add elements with id 'skipbx' and 'wrongbx' but use it early. You should fix that